OpenVMS Support was removed from gcc5-aux, so remove Name_AST_Entry
This patch can be applied to gcc-aux version too.

--- asis/a4g-a_sem.adb.orig	2014-04-10 13:38:35 UTC
+++ asis/a4g-a_sem.adb
@@ -1317,14 +1317,12 @@ package body A4G.A_Sem is
               Name_Complex_Representation       |
               Name_CPP_Class                    |
               Name_CPP_Constructor              |
-              Name_Export_Exception             |
               Name_Export_Function              |
               Name_Export_Object                |
               Name_Export_Procedure             |
               Name_Export_Valued_Procedure      |
               Name_Favor_Top_Level              |
               Name_Finalize_Storage_Only        |
-              Name_Import_Exception             |
               Name_Import_Function              |
               Name_Import_Object                |
               Name_Import_Procedure             |
@@ -1371,22 +1369,6 @@ package body A4G.A_Sem is
             end if;
 
          --  Cases when a specific processing is needed
-         when Name_Float_Representation =>
-            Pragma_Arg := First (Pragma_Argument_Associations (Pragma_Node));
-
-            if Present (Next (Pragma_Arg)) then
-               Pragma_Arg := Next (Pragma_Arg);
-            end if;
-
-            Pragma_Arg := Sinfo.Expression (Pragma_Arg);
-
-            if Entity (Pragma_Arg) = Entity_Node
-             or else
-               Chars (Pragma_Arg) = Chars (Entity_Node)
-            then
-               Result := True;
-            end if;
-
          when Name_Obsolescent =>
 
             if Is_Obsolescent (Entity_Node) then
@@ -1633,8 +1615,6 @@ package body A4G.A_Sem is
                if not (
                      N in First_Pragma_Name .. Last_Pragma_Name
                     or else
-                     N = Name_AST_Entry
-                    or else
                      N = Name_CPU
                     or else
                      N = Name_Interface
--- asis/a4g-mapping.adb.orig	2014-04-10 13:38:35 UTC
+++ asis/a4g-mapping.adb
@@ -1884,7 +1884,6 @@ package body A4G.Mapping is
             Attribute_Chars = Name_Address_Size             or else
             Attribute_Chars = Name_Asm_Input                or else
             Attribute_Chars = Name_Asm_Output               or else
-            Attribute_Chars = Name_AST_Entry                or else  -- VMS
             Attribute_Chars = Name_Bit                      or else
             Attribute_Chars = Name_Bit_Position             or else
             Attribute_Chars = Name_Code_Address             or else
@@ -3169,7 +3168,6 @@ package body A4G.Mapping is
 
       elsif Pragma_Chars in
               First_Pragma_Name .. Last_Pragma_Name |
-              Name_AST_Entry                        |
               Name_Interface
       then
          --  We have already checked for all the standard pragma names, so
